pkg/kubectl/cmd/stop.go

```diff
@@ -86,3 +63,24 @@ func NewCmdStop(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().Bool("all", false, "[-all] to select all the specified resources")
 	return cmd
 }
+
+func RunStop(f *cmdutil.Factory, cmd *cobra.Command, args []string, filenames util.StringList, out io.Writer) error {
+	cmdNamespace, err := f.DefaultNamespace()
+	if err != nil {
+		return err
+	}
+	mapper, typer := f.Object()
+	r := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
+		ContinueOnError().
+		NamespaceParam(cmdNamespace).RequireNamespace().
+		ResourceTypeOrNameArgs(false, args...).
+		FilenameParam(filenames...).
+		SelectorParam(cmdutil.GetFlagString(cmd, "selector")).
+		SelectAllParam(cmdutil.GetFlagBool(cmd, "all")).
+		Flatten().
+		Do()
+	if r.Err() != nil {
+		return r.Err()
+	}
+	return ReapResult(r, f, out, true)
```

Comment:
Shouldn't this be false? By running `stop` user is explicitly requesting reaper.